html/attributes.html

```diff
@@ -1,26 +1,31 @@
 <html>
+
 <body>
 
-<h1>The form autocomplete attribute</h1>
+    <h1>The form autocomplete attribute</h1>
+
+    <p>Fill in and submit the form, then reload the page, start to fill in the form again - and see how autocomplete
+        works.</p>
 
-<p>Fill in and submit the form, then reload the page, start to fill in the form again and see how autocomplete works.</p>
+    <p>Then, try to set autocomplete to "off".</p>
 
-<p>Then, try to set autocomplete to "off".</p>
+    <form action="/action_page.php" autocomplete="on">
```

Comment:
Question:
- Did you understood what dose this line of html ?

Reply:
yes,I did